src/test/resources/com/puppycrawl/tools/checkstyle/checks/trailingcomment/InputTrailingCommentPastFalsePositive.java

```diff
@@ -0,0 +1,129 @@
+package com.puppycrawl.tools.checkstyle.checks.trailingcomment;
+
+/**
+ * Config:
+ * default
+ */
+public class InputTrailingCommentPastFalsePositive {
```

Comment:
naming is not good, please change it to something else.

Reply:
fixed